steamworks/__init__.py

```diff
@@ -37,7 +37,7 @@ class STEAMWORKS(object):
     """
         Primary STEAMWORKS class used for fundamental handling of the STEAMWORKS API
     """
-    _arch = util.get_arch()
+    _arch = s_util.get_arch()
```

Comment:
Change to `steamworks_util` to reflect requested change

Reply:
Done - that sounds reasonable to me